packages/ui-kit/package.json

```diff
@@ -34,10 +34,15 @@
     "prism-react-renderer": "1.2.1",
     "prop-types": "15.8.1",
     "react-intl": "^5.7.0",
+    "react-is": "^17.0.2",
     "swr": "1.1.2",
     "tiny-invariant": "1.2.0"
   },
   "devDependencies": {
+    "@svgr/cli": "6.2.0",
+    "@types/react": "^17.0.38",
+    "@types/react-dom": "^17.0.11",
+    "@types/react-is": "^17.0.3",
```

Comment:
I would define them as `dependencies`. This way, consumers don't need to explicitly add them.

Reply:
Why I made them a dev dependency:

These data structures are not part of what the ui kit exorts so consumers don't need to know them and we can save consumers an unused dependency that they just have to maintain to silence yarn warnings.  

If a package exports an interface it should be a dependency, sure.

Reply:
The uikit package does export the type declarations actually. Anyway, not super important.